### PR TITLE
DOCS-1211-CE-About

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CE 3.17/next

Issue:
https://tigera.atlassian.net/browse/DOCS-1211

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Tests passed

### CODE REVIEW

File: calico-enterprise/about/index.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| Multi-cluster management                           | &bull; Unified management plane to manage cluster and workloads running on different infrastructure and using different Kubernetes distributions<br />&bull; Federated endpoints for policy-writing efficiency<br />&bull; Federated services to extend and automate endpoint sharing<br />&bull; Federated Prometheus metrics |
+| Logs and compliance reports                        | &bull; Out-of-the-box support for PCI DSS, SOC 2, HIPAA, GDPR, NIST, and custom frameworks<br />&bull; Out-of-the-box CIS benchmarks for Kubernetes compliance reports<br />&bull; Pre-defined and custom compliance reports for audit reporting (on-demand or scheduled)<br />&bull; Auditor-ready cluster compliance history |
```
Comment (girishshankaran) [inline calico-enterprise/about/index.mdx:25]: Logs and compliance reports

Perhaps we can move this to the end of the table (last entry)?

File: calico-enterprise/about/index.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+| Advanced Calico networking policy                  | &bull; Policy recommendations to isolate namespaces with network policy<br />&bull; Tiered policy<br />&bull; DNS policy <br /> &bull; Application layer policy with Envoy as daemonset<br />&bull; Auto host endpoints<br />&bull; Stage and preview impacts on traffic before enforcing policy<br />&bull; Network sets to reuse and scale sets of IP addresses used in policies<br />&bull; Policy integration with Panorama, Fortinet, and AWS firewalls |
+
+For a detailed list of {{prodname}} features, see [Tigera product comparison](../about/product-comparison.mdx)
+
+## Going into production with {{prodname}}
+
+It is not easy navigating the cultural shifts that come with adopting Kubernetes. Going into production with Kubernetes can seem an elusive goal. Tigera's **Customer Success** has spent many years working with enterprise companies in highly-regulated industries to understand the sticking points that stall adoption. Common hurdles for teams are:
```
Comment (girishshankaran) [inline calico-enterprise/about/index.mdx]: These two sentences (It is not easy.... and Going into.....) may not be true for enterprise customers that have successfully started their Kubernetes journey.

**Review by girishshankaran — APPROVED** [review]
LGTM. Two minor suggestions.

File: calico-enterprise/about/index.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/about/index.mdx]: The audience for this article are prospects that are not part of the "early adopter" profile.

**Pull request merged**